README.md

```diff
@@ -17,3 +17,11 @@ ommitted.)
 ![Deployed AWS resources and network diagram](./assets/dependency-track-aws-terraform.png "Deployed AWS Resources")
 
 The terraform is deployed using Companies House Concourse.
+
+## SSO Configuration
+
+SSO Is configured in both the API and Frontend via the environment variables.
+This configuration is stored as secrets within the vault and read into the
+applications via SSM Parameter Store.
+
+<!-- TODO: Flesh out with full SSO configuration once known -->
```

Comment:
What's tracking this TODO? Is it in your head over the next few days (OK) or is it longer term (Jira)?

Reply:
I'll add a JIRA task to the story, atm I don't know the final group structure which I want to document here so I left a Todo.

Reply:
RAND-366 relates